queue_job/job.py

```diff
@@ -532,14 +551,40 @@ def store(self):
             if self.channel:
                 vals.update({'channel': self.channel})
 
-            self.env[self.job_model_name].sudo().create(vals)
+            job = self.env[self.job_model_name].sudo().create(vals)
 
     def db_record(self):
         return self.db_record_from_uuid(self.env, self.uuid)
 
+    def _get_abs_context(self, original_ctx, ctx):
+        try:
+            import_ctx = json.loads(original_ctx)
+            current_ctx = json.loads(ctx)
+        except Exception as e:
+            _logger.error("\n\nERROR CONTEXT JSON CONVERSION: %s\n\n" % e)
```

Comment:
You were not successful with `ast` or `astor` like we spoke about on the Telegram?

Reply:
Ah yes - even `ast` or `astor` could not convert an Odoo object to a string properly. So let's forget about that.